.github/workflows/deploy_to_dev.yml

```diff
@@ -96,4 +96,11 @@ jobs:
           export TF_VAR_paas_app_docker_image=dfedigital/early-careers-framework-prod:${{ github.sha }}
           cd terraform/app
           terraform init -reconfigure -input=false -backend-config="bucket=paas-s3-broker-prod-lon-7f2ca242-9929-4662-a79c-c454ea56ea7b"
-          terraform apply -input=false -auto-approve -var-file ../workspace-variables/dev.tfvars -var='secret_paas_app_env_values={"RAILS_MASTER_KEY":"${{secrets.RAILS_MASTER_KEY_DEV}}"}' -var 'logstash_url=${{secrets.SYSLOG_DRAIN_URL}}' -var 'docker_username=${{ secrets.DOCKER_USERNAME }}' -var 'docker_password=${{ secrets.DOCKER_DEV_PASSWORD }}'
+          terraform apply \
+            -input=false \
+            -auto-approve \
+            -var-file ../workspace-variables/dev.tfvars \
+            -var='secret_paas_app_env_values={"RAILS_MASTER_KEY":"${{secrets.RAILS_MASTER_KEY_DEV}}","GOOGLE_BQ_SERVICE_ACCOUNT_BASE64":"${{secrets.GOOGLE_BQ_SERVICE_ACCOUNT_BASE64}}"}' \
```

Comment:
So, `GOOGLE_BQ_SERVICE_ACCOUNT_BASE64` is a secret that is only used inside the app?

So in principle, it could be put inside Rails credentials, and be accessible using `RAILS_MASTER_KEY` for a particular environment, right?

Reply:
Sure I can change. Although I'm not a fan of this. I believe the application code should be "configuationless" or "stateless" with these things being injected in. Otherwise code changes are needed for something which should be a configuration change. Now this has been blurred and less clean cut.

Reply:
Well, I think the other side of this spectrum is "if our secrets are held in github, then we need like 4 copies of them per secret, and our github gets littered very quickly". Which is not quite true here, cause we use the same secret for all envs.

I don't really mind which way it goes.

Reply:
Well we're using it anyway, in for a penny in for a pound.